scripts/catch/1.9.6/script.sh

```diff
@@ -0,0 +1,31 @@
+MASON_NAME=catch
+MASON_VERSION=1.9.6
+MASON_HEADER_ONLY=true
+
+. ${MASON_DIR}/mason.sh
+
+function mason_load_source {
+    mason_download \
+        https://github.com/philsquared/Catch/archive/v${MASON_VERSION}.tar.gz \
+        c8bf11d32c73c864ebf6b57e2a324a86ea39df82
+
+    mason_extract_tar_gz
+
+    export MASON_BUILD_PATH=${MASON_ROOT}/.build/catch-${MASON_VERSION}
+}
+
+# nothing to build, just copying single include header file
+function mason_compile {
+    mkdir -p ${MASON_PREFIX}/include/
+    cp -r single_include ${MASON_PREFIX}/include
```

Comment:
Done in https://github.com/mapbox/mason/pull/458/commits/73d8c0a25c79a099cb28f5214ff3a777d7a2bb26